benchmark/benchmark_N.py

```diff
@@ -0,0 +1,53 @@
+import time
+import math
+import socket
+from pathlib import Path
+
+import numpy as np
+import tensorflow as tf
+import matplotlib.pyplot as plt
+
+import utils
+import displace
+
```

Comment:
Can you please add some brief few-lines documentation describing what this script is doing, and also in other scripts. I understand it thanks to our zoom discussions, but just for the future quick orientation.